projects/pgai/pgai/vectorizer/embeddings.py

```diff
@@ -1,37 +1,16 @@
 import math
-import os
-import re
 import time
 from abc import ABC, abstractmethod
-from collections.abc import Awaitable, Callable, Mapping, Sequence
+from collections.abc import Awaitable, Callable, Sequence
 from dataclasses import dataclass
-from functools import cached_property
-from typing import (
-    Any,
-    Generic,
-    Literal,
-    TypeAlias,
-    TypeVar,
-)
-
-import ollama
-import openai
+from typing import Generic, TypeAlias, TypeVar
+
 import structlog
-import tiktoken
-import voyageai
-import voyageai.error
 from ddtrace import tracer
-from openai import resources
-from pydantic import BaseModel
-from typing_extensions import TypedDict, override
-
-MAX_RETRIES = 3
-
-TOKEN_CONTEXT_LENGTH_ERROR = "chunk exceeds model context length"
 
-openai_token_length_regex = re.compile(
-    r"This model's maximum context length is (\d+) tokens"
-)
+# Import needed to make the embedders available under this module, 
+# avoiding any breaking change after the split refactor.
+from .embedders import *  # noqa: F403
```

Comment:
Any idea on how to avoid this? My idea is to avoid adding a breaking change with this refactor as the import paths for the embedders are changing. 
If I don't add this import here, import paths for embedders such as Ollama will need to change from `from .embeddings import Ollama` to `from .embedders import Ollama`., causing a breaking change (and the need of releasing a major version)